src/index.html

```diff
@@ -10,5 +10,159 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <span><nav>HOME</nav></span>
+    <span><button>DOWNLOAD MY FULL RESUME</button></span>
+    <div><h1>John Roberts</h1></div>
+    <div><p>BUSINESS EXPERT</p></div>
+    <div>
+      <img src="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.jpg"
+           alt="John Roverts"
+      />
+    </div>
+    <span><button>DOWNLOAD MY FULL RESUME</button></span>
+    <div><h2>ABOUT ME</h2></div>
+    <span>
+      <button><img src="LinkedIn.jpg" alt="LinkedIn" /></button>
```

Comment:
It uses a relative path for the image source but can't find the image files submitted with the same PR.